packages/cursorless-vscode/src/ide/vscode/VscodeFileSystem.ts

```diff
@@ -1,52 +1,15 @@
-import {
-  Disposable,
-  FileSystem,
-  PathChangeListener,
-  walkFiles,
-} from "@cursorless/common";
-import { stat } from "fs/promises";
-import { max } from "lodash";
+import { Disposable, FileSystem, PathChangeListener } from "@cursorless/common";
+import { RelativePattern, Uri, workspace } from "vscode";
 
 export class VscodeFileSystem implements FileSystem {
   watchDir(path: string, onDidChange: PathChangeListener): Disposable {
-    // Just poll for now; we can take advantage of VSCode's sophisticated
-    // watcher later. Note that we would need to do a version check, as VSCode
-    // file watcher is only available in more recent versions of VSCode.
```

Comment:
Do we remember what versions this was referring to?

I suppose the types must exist if the type checker is happy, so I'm sure we're not dropping support for any version 😄

Reply:
Good call; I've bumped our min version to the one that supports watching arbitrary files. Note that the type system doesn't help us here, because it's the same api but it would fail at runtime if you tried to watch a file outside the user's current project